dashboard-ui/src/components/ActionsCompare/decisionList.jsx

```diff
@@ -86,196 +67,38 @@ class DecisionList extends React.Component {
-          "BAG_ACCESS",
-          "TOOL_HOVER",
-          "TOOL_SELECTED",
-          "TOOL_DISCARDED",
-          "TAG_SELECTED",
-          "BAG_CLOSED",
-          "SESSION_END",
-          "S_A_L_T_WAVED",
-          "S_A_L_T_WALK",
-          "S_A_L_T_WALK_IF_CAN",
-          "S_A_L_T_WAVE_IF_CAN",
-          "TOOL_APPLIED",
-          "TAG_DISCARDED",
-          ""
-        ];
-        return !excludedActionTypes.includes(row.actionType);
-      });
-  }
-
-  parseDoc() {
-    const { htmlFileContent } = this.state;
-    const parser = new DOMParser();
-    const doc = parser.parseFromString(htmlFileContent, 'text/html')
-
-    this.setState({ doc: doc })
-    this.parseTables(doc)
-  }
-
-  parseTables(doc) {
-    const tableTags = doc.getElementsByTagName('tbody')
-    let tableArray = Array.from(tableTags)
-    if (this.props.selectedScenario === "st-september-2023-mvp2") {
-      // have to filter out different actions from simulator based on scenario
-      tableArray = this.filterOutCasualtiesForSoarTech(utility.getCasualtyArray(tableArray))
-    } else if (this.props.selectedScenario === "adept-september-demo-scenario-1") {
-      tableArray = this.filterOutCasualtiesForBBN(utility.getCasualtyArray(tableArray))
-    }
-
-    this.setState({ casualtyArray: tableArray })
-    this.parseDecisions()
-  }
-
-  parseDecisions() {
-    let decisions = this.humanImageToDecisionMapping(this.state.csvFileContent, this.state.casualtyArray);
-    if (this.props.selectedScenario === "st-september-2023-mvp2") {
-      decisions = this.filterDecisionsSoarTech(decisions)
-    } else if (this.props.selectedScenario === "adept-september-demo-scenario-1") {
-      decisions = this.filterDecisionsBBN(decisions)
-    }
-    this.setState({ decisions: decisions })
-  }
-
-  // temporary filtering out bbn scenario and marine that ADM does not interact with
-  filterOutCasualtiesForSoarTech(casualties) {
-    // Names to be filtered out
-    const filteredNames = [
-      "Civilian Bob_4 Root",
-      "Military Mike Jungle Combat_1_5 Root",
-      "Military Mike Jungle Scout_1_3 Root"
-    ];
-
-    // Filter the casualties array based on the names
-    const filteredCasualties = casualties.filter(casualty => !filteredNames.includes(casualty.name));
-
-    return filteredCasualties;
-  }
-
-  filterOutCasualtiesForBBN(casualties) {
-    // Names to be kept
-    const filteredNames = [
-      "Asian Bob_4 Root",
-      "Military Mike Jungle Scout_1_3 Root"
-    ];
-
-    // Filter the casualties array based on the names
-    const filteredCasualties = casualties.filter(casualty => filteredNames.includes(casualty.name));
-
-    return filteredCasualties;
-  }
-
-
   filterDecisionsSoarTech(decisions) {
     // filter out decisions that involve bbn scenario 
-    const filteredNames = [
-      "Civilian Bob_4 Root",
-      "Military Mike Jungle Combat_1_5 Root",
-      "Military Mike Jungle Scout_1_3 Root"
+    const relevantCasualties = [
```

Comment:
At some point we will want this array and the array at line 95 to be dynamic and not hard coded in here.